README.md

```diff
@@ -42,13 +42,53 @@ npm install @elyra/pipeline-editor
 yarn add @elyra/pipeline-editor
 ```
 
-Or can be linked locally:
+Or can be built and linked locally:
 
 ```sh
 git clone git@github.com:elyra-ai/pipeline-editor.git
 cd pipeline-editor
 
-make clean install link
+npm run clean
+yarn install
+npm run build
+npm run link
+```
+
+Then in the project you're using the local build run the following:
+
+```sh
+yarn link @elyra/pipeline-services
+yarn link @elyra/pipeline-editor
+```
+
+If your project depends on `react` you need to link it back to prevent version conflicts
+
+Run in your project:
+
+```sh
+cd node_modules/react
+yarn link
+cd ../@types/react
+yarn link
+```
+
+or if you're running with Elyra you can use make:
+
+```sh
+make dev-link
+```
+
+Run in `pipeline-editor`
+
+```sh
+npm run dev-link
+npm run build
+```
+
+If you're connecting to a Jupyterlab extension run:
+
+```sh
+npm run jl-link
```

Comment:
Might be good to also include the build steps after this. Can you run `make clean install`? And what other steps might you need to install the JL extension with the linked package?

Reply:
Also - maybe unlink steps as well?

Reply:
Also watch?